### ISSUE TITLE
wc-reports display incorrect stock data on variable products

### ISSUE THREAD

**Issue by federicovezzoli** [woothemes/woocommerce#7160]
I see a problem with the wc-reports.
Under the tab of the out of stock products there are listed basically all the products in the shop.
These because the products are pretty much all variable products and the query is not working correctly.
In that case the out of stock reports give me a Zero but the product results available.
At line 119 of includes/admin/reports/class-wc-report-stock.php 
    
    case 'stock_level' :
    //echo $product->get_stock_quantity();
    echo $product->get_total_stock();
    break;

I changed the output number and is all good, but the rows get printed anyways, so I think the query needs to be fixed and I'm not good at it.

thanks

**Comment by mikejolley** [comment]
The query appears fine. Here it is for reference:

`
"FROM {$wpdb->posts} as posts
			INNER JOIN {$wpdb->postmeta} AS postmeta ON posts.ID = postmeta.post_id
			INNER JOIN {$wpdb->postmeta} AS postmeta2 ON posts.ID = postmeta2.post_id
			WHERE 1=1
			AND posts.post_type IN ( 'product', 'product_variation' )
			AND posts.post_status = 'publish'
			AND postmeta2.meta_key = '_manage_stock' AND postmeta2.meta_value = 'yes'
			AND postmeta.meta_key = '_stock' AND CAST(postmeta.meta_value AS SIGNED) <= '{$stock}'
		"
`

This is from 2.2/2.3.

Basically, get products and variations where manage_stock is YES and stock <= 0. In your case, the manage_stock property of your variable products must be incorrect, or turned on by accident. If you save a variable product you can see if they disappear from the report.  

If that is the case, the products may have been imported into your store incorrectly.

Closing this as I cannot reproduce an issue in my store, but let us know what you find.

**Comment by federicovezzoli** [comment]
yep, you're right.
It was a misconfiguration.

thanks for the feedback!
